emr.tf

```diff
@@ -162,11 +162,18 @@ resource "aws_emr_cluster" "hbase_read_replica" {
         "hbase.emr.readreplica.enabled": "true"
       }
     },
-{
+ {
     "Classification": "hdfs-site",
     "Properties": {
       "dfs.replication": "1"
     }
+  },
+ {
+    "classification": "core-site",
+    "properties": {
+      "fs.s3n.impl": "com.amazon.ws.emr.hadoop.fs.EmrFileSystem"
+    },
+    "configurations": []
   },
```

Comment:
Changes needed - EMR will adjust the config when launching the cluster, and it will not match the tf

classification -> Classification
properties -> Properties

remove "configurations" line